.rubocop_todo.yml

```diff
@@ -1,11 +1,27 @@
 # This configuration was generated by
-# `rubocop --auto-gen-config`
-# on 2020-10-30 22:23:14 UTC using RuboCop version 0.93.1.
+# `rubocop --auto-gen-config --auto-gen-only-exclude`
+# on 2020-11-03 23:10:27 UTC using RuboCop version 0.93.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 2
+# Configuration parameters: IgnoredMethods, Max.
+Metrics/AbcSize:
```

Comment:
I really like these inline, but if you prefer it here. 🤷

Reply:
I feel that the TODO file is fine for a legacy project where you use rubocop, but a greenfield project shouldn't have one.

Reply:
I have a slight preference for concentrating these into the todo file rather than sprinkled throughout the codebase.

Reply:
It's been my experience, that we never do the things in the TODO.  It's also not clear which areas you are currently working in are flagged for being ripe for refactoring.

Reply:
Do you think this might make for a good dev planning discussion?